app/components/support_form_component.html.erb

```diff
@@ -55,15 +54,27 @@
             </dd>
           </div>
         <% end %>
+
+        <% if cohort_year.present? %>
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+              Cohort
```

Comment:
I think we should use "academic year" rather than "cohort" - as I think we've moved away from using the cohort term in the school interface?

```suggestion
              Academic year
```

Reply:
The academic year should be presented as '20XX to 20XX'. Thanks